app/javascript/packs/application.js

```diff
@@ -3,12 +3,10 @@
 // a relevant structure within app/javascript and only use these pack files to reference
 // that code so it'll be compiled.
 
-require('@rails/ujs').start()
 require('turbolinks').start()
-require('@rails/activestorage').start()
-require('channels')
-require('trix')
-require('@rails/actiontext')
+require('alpinejs')
```

Comment:
This seems to be very disruptive: https://github.com/alpinejs/alpine#use
_What are the benefits of using this? Remember the more tools we have more layers of complexity to overcome in order to contribute to the project_ (nevertheless, it doesn't mean it is not worth it, but I want to know the why on this decision)

Reply:
This is an easy approach to keep everything in the template. For things like closing modals click events and validations works good. I'm trying together with Tailwind UI and it works great.